mixins/localize-dynamic-mixin.js

```diff
@@ -7,6 +7,12 @@ export const LocalizeDynamicMixin = superclass => class extends LocalizeMixin(su
 
 	static async getLocalizeResources(langs, { importFunc, osloCollection }) {
 
+		// in dev, don't request unsupported langpacks
+		if (!importFunc.toString().includes('switch')) {
```

Comment:
Seems _slightly_ fragile... maybe a couple unit tests to cover both cases so we don't accidentally break this in the future?